.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+
+---
+# [Template] PR Description
```

Comment:
Why are there 2 PR templates? The second one seems more appropriate for FauxPilot. What do you think?